src/tutorials.js

```diff
@@ -5,6 +5,14 @@ const moment = require('moment');
 const axiosPath = require('./axiosPath');
 
 const tutorials = [
+  {
+    title: 'How to Use the void operator',
```

Comment:
We already have a tutorial about the void operator. This is supposed to be a quick tutorial about `void(0)` and how `void(0)` is not actually a function call.